### PR TITLE
Add-Error-Message-To-Unify-Not-Dir

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/38351

## Description
When giving input to unify that is a path to a file (e.g, YML of integration) it does not indicate that the input is not expected but sends an unclear error message.
Enhanced the command to return a more precise error message.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
unit test failure before addition
https://app.circleci.com/pipelines/github/demisto/demisto-sdk/10036/workflows/0956f5a9-7293-4ae8-a492-0eb5bf17d4a9/jobs/46255

File: demisto_sdk/commands/unify/unifier.py [hunk 1]
```diff
@@ -52,6 +53,10 @@ def __init__(self, input: str, dir_name=INTEGRATIONS_DIR, output: str = '',
         # Changing relative path to current abspath fixed problem with default output file name.
         if input == '.':
             input = os.path.abspath(input)
+        if not os.path.isdir(input):
+            print_error('You have failed to provide a legal file path, a legal file path '
+                        'should be to a directory if an integration or script.')
```
Comment (dantavori) [inline demisto_sdk/commands/unify/unifier.py:58]:
```suggestion
                        'should be to a directory of an integration or a script.')
```

File: demisto_sdk/commands/unify/tests/unifier_test.py [hunk 2]
```diff
@@ -81,8 +81,10 @@ def __init__(self, tenant_id: str = '', auth_id: str = '', enc_key: str = '',
 TESTS_DIR = f'{git_path()}/demisto_sdk/tests'
 
 
-def test_clean_python_code():
-    unifier = Unifier("test_files/VulnDB")
+def test_clean_python_code(repo):
+    pack = repo.create_pack('PackName')
+    integration = pack.create_integration('integration', 'bla', INTEGRATION_YAML)
+    unifier = Unifier(str(integration.path))
```
Comment (dantavori) [inline demisto_sdk/commands/unify/tests/unifier_test.py:87]: why did you change this?

Comment (tomneeman151293) [reply demisto_sdk/commands/unify/tests/unifier_test.py:87]: This path did not really exists, thats why os.path.isdir failed. so I created a mock integration just for the path to indeed exist.

**Review by dantavori — APPROVED** [review]
